tests/resources/Vsphere-Util.robot

```diff
@@ -234,3 +234,14 @@ Check VM Guestinfo
     ${rc}  ${output}=  Run And Return Rc And Output  govc vm.info -e ${vm} | grep ${str}
     Should Be Equal As Integers  ${rc}  0
     [Return]  ${output}
+
+Clean VMOMI Sessions
+    [Arguments]  ${user-agent}
+    Run  for x in $(govc session.ls); grep "${user-agent}" | xargs govc session.rm
```

Comment:
From a quick glance, I am thinking this is way too open.  I just took a quick look at govc session.ls and session.rm and I am not very familiar with it, but I am imagining this is going to wipe out any session related to vic-machine on the host, which would include other tests that might be running in parallel?